.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JavaScriptSettings">
+    <option name="languageLevel" value="FLOW" />
```

Comment:
If i recall correctly this is configured for "flow" (ts alternative) which we don't use. Do you know how this was generated?

Reply:
No, but I will remove it.